jira/util/reports_generation/README.md

```diff
@@ -1,16 +1,19 @@
-## CSV report aggregator and chart generation tool
+## Reports generator - tool that creates aggregated .csv files and charts
 Before you start, make sure you have installed Python packages from `jira/requirements.txt`.
 
 Otherwise, run the `pip install -r requirements.txt` command from `jira` directory to install necessary packages to your virtual environment.
 
-To create aggregated csv report and chart with one command (recommended), run the <br> 
+To create reports, run the <br>
 `python csv_chart_generator.py [performance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation` folder.
 
-To create aggregated csv report or chart separately:
-* To create only an aggregated csv report, run the <br>
-`python csv_aggregator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/csv_aggregator` folder.
+The aggregated .csv files and charts are stored in the `jira/results/reports` directory.
+Before run, you should edit `performance_profile.yml` or `scale_profile.yml` and set appropriate `fullPath` values. 
 
-* To create only a chart based on already created aggregated csv report, run the <br>
-`python csv_chart_generator.py [perfomance_profile.yml or scale_profile.yml]` command from the `jira/util/reports_generation/chart_generator` folder.
-
-The aggregated csv report and chart are stored in the jira/util/reports_generation/results directory.
+**Configuration**
+- `column_name` - means column from `results.csv` to comparison.
```

Comment:
- `column_name` - column name from `results.csv` used for aggregation
- `run_name` - label for specific run
- `fullPath` -  the full path to result folder of specific run
- `index_col` - index column
- `title` - chart title (also this value is used as file name)
- `image_height_px` - chart image height in pixels
- `image_width_px` - chart image width in pixels

Reply:
done